### DISCUSSION TITLE
How to add your own csv file to allow chat on the documentation

### DISCUSSION THREAD

**Discussion by sharadkhoji123** [mythrantic/ollama-docker#12] [General]
When I use ollama on my local system, I can pass my csv and chat with it using below piece of code. But how can I do that here ?
from pandasai import SmartDataframe
from langchain_community.llms import Ollama
llm = Ollama(base_url='http://localhost:11434', model='llama3:8b')
df = SmartDataframe("dummy01.csv", config={"llm": llm})
response = df.chat('Which is the highest selling product ?')
print(response)

**Comment by valiantlynx** [comment]
Check out the file src/rag.py in the example app code. It's just langchain. So whatever is in the langchain docs works
